src/helpers/validate.js

```diff
@@ -9,7 +9,7 @@ function validateURL(url) {
 }
 
 function validateFlags(flags) {
-  const mask = /^(?!-)[\w,-]+$/
+  const mask = /^[\w,\.,\-]{1,45}$/
```

Comment:
There is benefit in moving this regex outside.

Currently, every invocation of `validateFlags` will instantiate a unique regular expression instance which will promptly be destroyed (and never reused) after the function returns. The optimization here is to simply instantiate one regular expression but share it among every invocation of the function. This is common practice with regular expressions in JavaScript because complex patterns can be (relatively) difficult to compile.

Reply:
Is this intended to match commas?

Reply:
```suggestion
  const mask = /^[\w\.\-]{1,45}$/
```
fix: do not match commas

Reply:
> Currently, every invocation of validateFlags will instantiate a unique regular expression instance which will promptly be destroyed (and never reused) after the function returns

I don't follow. Except in tests, this regex should generally only be used once per execution